### DISCUSSION TITLE
支持一个默认的用户组可以拥有项目内所有流水线的权限

### DISCUSSION THREAD

**Discussion by cxvdzxhb** [Tencent/bk-ci#4223] [General]
蓝盾现有的权限体系中，权限控制粒度太细了。蓝盾默认支持的几种用户组中，要么管理员拥有所有权限，要么其他用户组几乎没有权限，在实际使用过程中，开发同学经常要申请蓝盾的各种权限，如：项目组权限、流水线权限、codecc任务查看权限，阻碍了研发效率。在**业务团队**开发的角度，相同业务的仓库会放在同一个group，同一个group的仓库对应的流水线也一般会放在同一个蓝盾项目；开发同学的日常开发工作是在一个（或者多个）group的多个仓库进行开发，这些仓库（及对应流水线）有的可能不是自己创建的，按照蓝盾的权限体系，开发同学不是管理员的话，查看执行其他同学创建的流水线是需要申请权限的。
我希望蓝盾提供一个用户组提供必要权限方便开发同学的开发工作，如：可以拥有项目内所有流水线的一切权限、及codecc任务等其他插件的查看、执行权限，但是没有质量红线的修改权限。

**Comment by zanyzhao** [comment]
> 如：可以拥有项目内所有流水线的一切权限、及codecc任务等其他插件的查看、执行权限，但是没有质量红线的修改权限。

目前蓝盾在腾讯内部对接的权限体系是“蓝鲸权限中心”，已经支持“自定义组”和“按组授权”，可以满足此类需求。
